debug.log

```diff
@@ -0,0 +1,16 @@
+[0601/122027.469:ERROR:process_reader_win.cc(123)] NtOpenThread: {Access Denied} A process has requested access to an object, but has not been granted those access rights. (0xc0000022)
```

Comment:
This file does not need to commit. 
Add a .gitignore file and ignore this file.

check how to remove committed files